components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2159,6 +2159,9 @@ type Identity struct {
 	PrimaryEmail string   `json:"primaryEmail,omitempty"`
 	Readonly     bool     `json:"readonly,omitempty"`
 	Tokens       []*Token `json:"tokens,omitempty"`
+
+	// The last time this entry was touched during a signin.
+	LastSigninTime time.Time `json:"last_signin_time,omitempty"`
```

Comment:
Does this actually work without transforming it to a string? Server uses ISO 8601 as the date format, while go will serialize into RFC 3339

Reply:
`2020-02-08 09+07:00` is valid ISO 8601 but not valid RFC 3339

Reply:
Yes, not everything is easy.